l1-contracts/contracts/zksync/facets/Executor.sol

```diff
@@ -472,4 +496,90 @@ contract ExecutorFacet is Base, IExecutor {
     function _setBit(uint256 _bitMap, uint8 _index) internal pure returns (uint256) {
         return _bitMap | (1 << _index);
     }
+
+    /// @notice Calls the point evaluation precompile and verifies the output
+    /// Verify p(z) = y given commitment that corresponds to the polynomial p(x) and a KZG proof.
+    /// Also verify that the provided commitment matches the provided versioned_hash.
+    /// 
+    function _pointEvaluationPrecompile(
+        bytes32 _versionedHash,
+        bytes32 _openingPoint,
+        bytes calldata _openingValueCommitmentProof
+    ) internal view {
+        bytes memory precompileInput = abi.encodePacked(_versionedHash, _openingPoint, _openingValueCommitmentProof);
+
+        (bool success, bytes memory data) = POINT_EVALUATION_PRECOMPILE_ADDR.staticcall(precompileInput);
+
+        require(success, "failed to call point evaluation precompile");
+        (, uint256 result) = abi.decode(data, (uint256, uint256));
+        require(result == BLS_MODULUS, "precompile unexpected output");
```

Comment:
Can not guess what it means

Reply:
I’ll add a comment but a successful call to the precompile returns the `BLS_MODULUS` https://eips.ethereum.org/EIPS/eip-4844#point-evaluation-precompile